### DISCUSSION TITLE
Particles position and velocity is single precision or double precision.

### DISCUSSION THREAD

**Discussion by liangt1995** [glotzerlab/hoomd-blue#1967] [General]
Hi community:
      
    I want to ask what is the format of particles position and velocity during running, single precision or double precision. I want to update one particle's position like "-80.0 + 0.00000001 * timestep" or keep its velocity as 0.000002 \sigma/\tau(using nve and zero_force). 

    I am using the double precision version of hoomd-blue 2.9.7。I found the position can not update right or the velocity can not work right. I wonder if it is the precision problem. 

   My hoomd version may be old,  but if it can work in new version, please let me konw.

  Thank you very much.

**Comment by joaander** [comment]
Please edit your post to reformat the content. Scrolling several page widths to read a single sentence is an utter waste of my time.

**Reply by liangt1995** [reply]
Sorry about that, I have reedit it in new discussion.

**Reply by joaander** [reply]
Thanks for spamming the board with extra messages. I said **EDIT** for a reason.
